Common/DTO/BrokerRequests/InternalLoginRequest.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace DTO.RestRequests
+{
+    public class InternalLoginRequest
```

Comment:
Why don;t use here UserCredentials

Reply:
Because UserCredential contains an Id and userId, but initially they are not on client.